File: BackEnd/eMenka/eMenka.Domain/Classes/Country.cs
using System.Collections.Generic;

namespace eMenka.Domain.Classes
{
    public class Country
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Code { get; set; }
        public string Nationality { get; set; }
        public bool POD { get; set; }
        public bool IsActive { get; set; }
        public IEnumerable<City> Cities { get; set; }
    }
}

File: eFenkaFrontend/src/app/models/country/country.ts
export class Country {
  id: number;
  name: string;
  code: string;
  nationality: string;
  POD: boolean;
  isActive: boolean;

  constructor(id: number, name: string, code: string, nationality: string, POD: boolean, isActive: boolean){
    this.id = id;
    this.name = name;
    this.code = code;
    this.nationality = nationality;
    this.POD = POD;
    this.isActive = isActive;
  }
}
